content/docs/_index.md

```diff
@@ -1,3 +1,6 @@
 ---
-title: stuff
+title: crossplane
+weight: 401
+layout: redirect
+to: https://crossplane.io/docs/v1.9/
```

Comment:
Will we need to manually update this on new versions being synced?

Reply:
That's the next PR coming. Putting it in place as a temp fix.